src/stdlib_ansi_cursor.f90

```diff
@@ -0,0 +1,107 @@
+module stdlib_ansi_cursor
+    use stdlib_strings, only: to_string
+    implicit none
+
+    character(len=*), parameter :: esc = achar(27)
+    !> moves the cursor to home => `(0,0)`
+    character(len=*), parameter :: home = esc//"[H"
+    !> erases from the cursor till the end of the screen
+    character(len=*), parameter :: clear_till_screen_end = esc//"[OJ"
+    !> erases from the cursor to the beginning of the screen
+    character(len=*), parameter :: clear_till_screen_start = esc//"[1J"
+    !> erases the entire screen
+    character(len=*), parameter :: clear_completely = esc//"[2J"
+    !> erases from the cursor till the end of line
+    character(len=*), parameter :: clear_till_line_end = esc//"[0K"
+    !> erases from the cursor till the beginning of the line
+    character(len=*), parameter :: clear_till_line_start = esc//"[1K"
+    !> erases the entire line
+    character(len=*), parameter :: clear_entire_line = esc//"[2K"
+
+contains
+    !> moves the cursor to `(line, column)`
+    !> returns an empty string if any of them is negative
+    pure function move_to(line, col) result(str)
```

Comment:
Could some of these functions be `elemental`?

Reply:
They could be made so, But I don't think it's worth it since I don't see anyone using it like that